js/lib/beautify-html.js

```diff
@@ -1034,4 +1052,4 @@
         };
     }
 
-}());
\ No newline at end of file
+}());
```

Comment:
You must run `./build` to ensure your code meets style guidelines for the project.